tools/llvm-spirv/CMakeLists.txt

```diff
@@ -1,15 +1,20 @@
-set(LLVM_LINK_COMPONENTS
-  Analysis
-  BitReader
-  BitWriter
-  Core
-  SPIRVLib
-  Support
-  TransformUtils
-  )
-
-include_directories(${LLVM_SPIRV_INCLUDE_DIRS})
-
-add_llvm_tool(llvm-spirv
-  llvm-spirv.cpp
-  )
+ADD_EXECUTABLE(llvm-spirv
```

Comment:
Why not use add_llvm_tool function?

Reply:
because it doesn't really work if used outside a full LLVM build

Reply:
also it hardcodes "bin" as the install target and doesn't use GNUInstallDirs, which... is annoying.